src/coffee/directives/search-box.coffee

```diff
@@ -38,6 +38,9 @@ angular.module('uiGmapgoogle-maps')
       
       link: (scope, element, attrs, mapCtrl) =>
         GoogleMapApi.then (maps) =>
+          unless scope.template?
```

Comment:
Another option is to set a default template that's just and HTML input field. What do you think?